http_prompt/printer.py

```diff
@@ -0,0 +1,20 @@
+import click
+
+
+class Printer():
```

Comment:
@eliangcs , no problem, it's already using `click.echo_via_pager`, I was just stressing it out in case there would be better solution and I haven't known about it... never mind...